src/components/Books.js

```diff
@@ -18,13 +18,13 @@ function Books() {
 
   return (
     <>
-      <div className="book-container">
+      <div className="container-fluid d-flex-column w-80 mt-5 book-container">
         {Object.keys(library.Booklist).map((bookItem) => (
 
-          <div className="book" key={uuidv4()}>
+          <div className="d-flex mt-3 book" key={uuidv4()}>
```

Comment:
- [ ] like the previous reviewer noted, Kindly add all the missing parts such as the progress bar so it matches the Zeplin design 👍🏽.